interfaces/builtin/media_hub_test.go

```diff
@@ -0,0 +1,208 @@
+		SlotInfo: &snap.SlotInfo{
+			Snap: &snap.Info{
+				SuggestedName: "media-hub",
+				Apps: map[string]*snap.AppInfo{"app1": app1,
+					"app2": app2},
+			},
+			Name:      "media-hub",
+			Interface: "media-hub",
+			Apps:      map[string]*snap.AppInfo{"app1": app1, "app2": app2},
+		},
+	}
+	release.OnClassic = false
+
+	apparmorSpec := &apparmor.Specification{}
+	err := apparmorSpec.AddConnectedPlug(s.iface, s.plug, slot)
+	c.Assert(err, IsNil)
+	c.Assert(apparmorSpec.SecurityTags(), DeepEquals, []string{"snap.other.app"})
+	c.Assert(apparmorSpec.SnippetForTag("snap.other.app"), testutil.Contains,
+		`peer=(label="snap.media-hub.*"),`)
+}
+
+// The label uses alternation when some, but not all, apps is bound to the media-hub slot
+func (s *MediaHubInterfaceSuite) TestConnectedPlugSnippetUsesSlotLabelSome(c *C) {
+	app1 := &snap.AppInfo{Name: "app1"}
+	app2 := &snap.AppInfo{Name: "app2"}
+	app3 := &snap.AppInfo{Name: "app3"}
+	slot := &interfaces.Slot{
+		SlotInfo: &snap.SlotInfo{
+			Snap: &snap.Info{
+				SuggestedName: "media-hub",
+				Apps: map[string]*snap.AppInfo{"app1": app1,
+					"app2": app2,
+					"app3": app3},
+			},
+			Name:      "media-hub",
+			Interface: "media-hub",
+			Apps:      map[string]*snap.AppInfo{"app1": app1, "app2": app2},
+		},
+	}
+	release.OnClassic = false
+
+	apparmorSpec := &apparmor.Specification{}
+	err := apparmorSpec.AddConnectedPlug(s.iface, s.plug, slot)
+	c.Assert(err, IsNil)
+	c.Assert(apparmorSpec.SecurityTags(), DeepEquals, []string{"snap.other.app"})
+	c.Assert(apparmorSpec.SnippetForTag("snap.other.app"), testutil.Contains,
+		`peer=(label="snap.media-hub.{app1,app2}"),`)
+}
+
+// The label uses short form when exactly one app is bound to the media-hub slot
+func (s *MediaHubInterfaceSuite) TestConnectedPlugSnippetUsesSlotLabelOne(c *C) {
+	app := &snap.AppInfo{Name: "app"}
+	slot := &interfaces.Slot{
+		SlotInfo: &snap.SlotInfo{
+			Snap: &snap.Info{
+				SuggestedName: "media-hub",
+				Apps:          map[string]*snap.AppInfo{"app": app},
+			},
+			Name:      "media-hub",
+			Interface: "media-hub",
+			Apps:      map[string]*snap.AppInfo{"app": app},
+		},
+	}
+	release.OnClassic = false
+
+	apparmorSpec := &apparmor.Specification{}
+	err := apparmorSpec.AddConnectedPlug(s.iface, s.plug, slot)
+	c.Assert(err, IsNil)
+	c.Assert(apparmorSpec.SecurityTags(), DeepEquals, []string{"snap.other.app"})
+	c.Assert(apparmorSpec.SnippetForTag("snap.other.app"), testutil.Contains,
+		`peer=(label="snap.media-hub.app"),`)
+}
+
+func (s *MediaHubInterfaceSuite) TestConnectedPlugSnippetAppArmor(c *C) {
+	apparmorSpec := &apparmor.Specification{}
+
+	err := apparmorSpec.AddConnectedPlug(s.iface, s.plug, s.slot)
+	c.Assert(err, IsNil)
+	c.Assert(apparmorSpec.SecurityTags(), DeepEquals, []string{"snap.other.app"})
+	c.Assert(apparmorSpec.SnippetForTag("snap.other.app"), Not(IsNil))
+	c.Assert(apparmorSpec.SnippetForTag("snap.other.app"), testutil.Contains,
+		`#include <abstractions/dbus-session-strict>`)
```

Comment:
This isn't specific to the plug side. Can you add another assert for a string that is specific to only the plug side?